src/stream.c

```diff
@@ -153,6 +153,9 @@ avifBool avifROStreamSkipBits(avifROStream * stream, size_t bitCount)
         const size_t padding = AVIF_MIN(8 - stream->numUsedBitsInPartialByte, bitCount);
         stream->numUsedBitsInPartialByte = (stream->numUsedBitsInPartialByte + padding) % 8;
         bitCount -= padding;
+        if (stream->numUsedBitsInPartialByte != 0) {
```

Comment:
I think `bitCount == 0` would be easier to understand but that is the proper fix.

Reply:
Thanks for the suggestion. It is also more efficient (avoids unnecessary work) if `stream->numUsedBitsInPartialByte == 0` and `bitCount == 0`. I already have a test for that condition in this pull request.

I switched to this test in https://github.com/AOMediaCodec/libavif/pull/2499.